src/psyclone/gocean1p0.py

```diff
@@ -80,25 +80,29 @@
 VALID_STENCIL_NAMES = ["go_pointwise"]
 
 # A dictionary giving the mapping from meta-data names for
-# properties of the grid to their names in the Fortran grid_type.
-GRID_PROPERTY_DICT = {"go_grid_area_t": "area_t",
-                      "go_grid_area_u": "area_u",
-                      "go_grid_area_v": "area_v",
-                      "go_grid_mask_t": "tmask",
-                      "go_grid_dx_t": "dx_t",
-                      "go_grid_dx_u": "dx_u",
-                      "go_grid_dx_v": "dx_v",
-                      "go_grid_dy_t": "dy_t",
-                      "go_grid_dy_u": "dy_u",
-                      "go_grid_dy_v": "dy_v",
-                      "go_grid_lat_u": "gphiu",
-                      "go_grid_lat_v": "gphiv",
-                      "go_grid_dx_const": "dx",
-                      "go_grid_dy_const": "dy",
-                      "go_grid_x_min_index": "subdomain%internal%xstart",
-                      "go_grid_x_max_index": "subdomain%internal%xstop",
-                      "go_grid_y_min_index": "subdomain%internal%ystart",
-                      "go_grid_y_max_index": "subdomain%internal%ystop"}
+# properties of the grid to their names and type in the Fortran grid_type.
```

Comment:
Perhaps add:

    # i.e. they can be accessed in Fortran by doing grid%<name>, e.g. grid%area_t or grid%subdomain%internal%xstart.

Reply:
Done, and the bug that I also fixed is in my comment at the end of this thread.